CHANGELOG.rst

```diff
@@ -1,30 +1,141 @@
+0.9.0
+=====
+
+This release contains a major overhaul of Anonlink’s API.
```

Comment:
Mention the headline feature *multiparty support*

Reply:
Done!